docs/src/documentation/03-components/03-information/alert/03-accessibility.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/alert/accessibility/
+---
+
+# Accessibility
+
+## Alert
+
+The Alert component has been designed with accessibility in mind.
+
+### Accessibility props
+
+The following props are available to improve the accessibility of your Alert component:
+
+| Prop       | Type     | Description                                                              |
+| :--------- | :------- | :----------------------------------------------------------------------- |
+| labelClose | `string` | Defines accessible text for the close button when the alert is closable. |
+
+When the `closable` prop is set to `true`, the `labelClose` prop is required. It provides the text that screen readers will announce when focusing on the close button, helping users understand its purpose.
+
+The `labelClose` text should be properly translated according to the user's language to ensure accessibility for all users, regardless of their preferred language.
+
+### Best practices
+
+- When using the `icon` prop, follow the accessibility guidelines documented in the Icon component's accessibility documentation to ensure that icons are properly accessible or visually hidden from screen readers when they are purely decorative.
+
+### Code examples
+
+#### Using labelClose with closable alert
+
+```jsx
+<Alert
+  type="info"
+  title="Notification"
+  closable
+  labelClose="Close notification"
+  onClose={() => {
+    // handle close action
+  }}
+>
+  This is an important update for your upcoming flight.
+</Alert>
+```
+
+In this example, screen readers would announce "Close notification" when focusing on the close button.
+
+#### Non-closable alert (no accessibility props required)
+
+```jsx
+<Alert type="success" title="Success">
+  Your booking has been confirmed.
+</Alert>
+```
+
+In this example, the alert doesn't require additional accessibility props as it doesn't have interactive elements that need labels.
+
+## AlertButton
```

Comment:
I wonder whether we should point to the Button a11y docs in this section, what do you think?